build.py

```diff
@@ -371,6 +375,9 @@ def RunTests(context,extraArgs):
 parser.add_argument("--debug-python", dest="debug_python", action="store_true",
                       help="Define Boost Python Debug if your Python library comes with Debugging symbols (default: %(default)s).")
 
+parser.add_argument("--qt-location", type=str,
+                    help="Directory where Qt is installed.")
+
```

Comment:
To build the new UI (which uses) Qt you need to specify where to find Qt (should be the same one that Maya is using).